test/Dockerfile

```diff
@@ -1,22 +1,18 @@
-FROM python:3.5.2
+FROM hashicorp/terraform as terraform
 
-ENV TERRAFORM_VERSION 0.11.8
+FROM python:3.6
+
+ENV TERRAFORM_VERSION 0.12.16
 ENV NODEJS_VERSION 8
```

Comment:
Is there any need for this line? If not please remove it.

Reply:
I have removed the `NODEJS_VERSION 8` line.